plugins/autosuspend.koplugin/main.lua

```diff
@@ -50,6 +50,12 @@ function AutoSuspend:_enabledShutdown()
     return Device:canPowerOff() and self.autoshutdown_timeout_seconds > 0
 end
 
+function AutoSuspend:_updateLastAction()
```

Comment:
Do you plan to remove the function after testing, as this is basically a one liner?

Reply:
Yes, definitely (as well as getting rid of the accompanying logging ;)) ;).